### ISSUE TITLE
erase duplicate types in unions

### ISSUE THREAD

**Issue by ctaggart** [fable-compiler/ts2fable#25]
From #19. This is due to all of the TypeScript unions that use enum cases.

**Comment by alfonsogarciacaro** [comment]
I don't understand exactly this use case, could you give an example?

**Comment by ctaggart** [comment]
There are several examples in typescript.fs. An example is CommentKind:

``` ts
type CommentKind = SyntaxKind.SingleLineCommentTrivia | SyntaxKind.MultiLineCommentTrivia;
```

It currently is generated as:
``` fs
    type CommentKind =
        U2<SyntaxKind, SyntaxKind>
```
but should just be:
``` fs
    type CommentKind =
        SyntaxKind
```

**Comment by alfonsogarciacaro** [comment]
Ah, I see now. You're totally right :+1: Thanks a lot for the clarification!

**Comment by ctaggart** [comment]
fixed by #31
